client/templates/minutes/minutesedit.js

```diff
@@ -487,23 +487,40 @@ Template.minutesedit.events({
                 }, 500);
             };
 
-            if (GlobalSettings.isEMailDeliveryEnabled()) {
-                ConfirmationDialogFactory.makeSuccessDialogWithTemplate(
-                    doFinalize,
-                    'Confirm finalize minutes',
-                    'confirmationDialogFinalize',
-                    {
-                        minutesDate: aMin.date,
-                        hasOpenActionItems: aMin.hasOpenActionItems(),
-                        sendActionItems: (sendActionItems) ? 'checked' : '',
-                        sendInformationItems: (sendInformationItems) ? 'checked' : ''
-                    },
-                    'Finalize'
+            let processFinalize = function(){
+                if (GlobalSettings.isEMailDeliveryEnabled()) {
+                    ConfirmationDialogFactory.makeSuccessDialogWithTemplate(
+                        doFinalize,
+                        'Confirm finalize minutes',
+                        'confirmationDialogFinalize',
+                        {
+                            minutesDate: aMin.date,
+                            hasOpenActionItems: aMin.hasOpenActionItems(),
+                            sendActionItems: (sendActionItems) ? 'checked' : '',
+                            sendInformationItems: (sendInformationItems) ? 'checked' : ''
+                        },
+                        'Finalize'
+                    ).show();
+                } else {
+                    doFinalize();
+                }
+            };
+
+            let noParticipantsPresent = true;
+            aMin.participants.forEach(p => {if(p.present == true) noParticipantsPresent = false;});
```

Comment:
I think it is better to check boolean values without a comparison with true or false: simply `if (p.present) { ... }`. A comparison with only two equal-signs can often lead to unwanted errors in JavaScript. Plus, codeclimate don't likes them which leads to bad grades :-(